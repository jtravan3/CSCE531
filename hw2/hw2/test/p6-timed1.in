x=1
x
(#2+#2)%101
(#3+#3)%101
(#4+#4)%101
(#5+#5)%101
(#6+#6)%101
(#7+#7)%101
(#8+#8)%101
(#9+#9)%101
(#10+#10)%101
(#11+#11)%101
(#12+#12)%101
(#13+#13)%101
(#14+#14)%101
(#15+#15)%101
(#16+#16)%101
(#17+#17)%101
(#18+#18)%101
(#19+#19)%101
(#20+#20)%101
(#21+#21)%101
(#22+#22)%101
(#23+#23)%101
(#24+#24)%101
(#25+#25)%101
(#26+#26)%101
(#27+#27)%101
(#28+#28)%101
(#29+#29)%101
(#30+#30)%101
(#31+#31)%101
(#32+#32)%101
(#33+#33)%101
(#34+#34)%101
(#35+#35)%101
(#36+#36)%101
(#37+#37)%101
(#38+#38)%101
(#39+#39)%101
(#40+#40)%101
(#41+#41)%101
(#42+#42)%101
(#43+#43)%101
(#44+#44)%101
(#45+#45)%101
(#46+#46)%101
(#47+#47)%101
(#48+#48)%101
(#49+#49)%101
x=3
#50
(#52+#52)%101
(#53+#53)%101
(#54+#54)%101
(#55+#55)%101
(#56+#56)%101
(#57+#57)%101
(#58+#58)%101
(#59+#59)%101
(#60+#60)%101
(#61+#61)%101
(#62+#62)%101
(#63+#63)%101
(#64+#64)%101
(#65+#65)%101
(#66+#66)%101
(#67+#67)%101
(#68+#68)%101
(#69+#69)%101
(#70+#70)%101
(#71+#71)%101
(#72+#72)%101
(#73+#73)%101
(#74+#74)%101
(#75+#75)%101
(#76+#76)%101
(#77+#77)%101
(#78+#78)%101
(#79+#79)%101
(#80+#80)%101
(#81+#81)%101
(#82+#82)%101
(#83+#83)%101
(#84+#84)%101
(#85+#85)%101
(#86+#86)%101
(#87+#87)%101
(#88+#88)%101
(#89+#89)%101
(#90+#90)%101
(#91+#91)%101
(#92+#92)%101
(#93+#93)%101
(#94+#94)%101
(#95+#95)%101
(#96+#96)%101
(#97+#97)%101
(#98+#98)%101
(#99+#99)%101
(#100+#100)%101
(#101+#101)%101
(#102+#102)%101
(#103+#103)%101
(#104+#104)%101
(#105+#105)%101
(#106+#106)%101
(#107+#107)%101
(#108+#108)%101
(#109+#109)%101
