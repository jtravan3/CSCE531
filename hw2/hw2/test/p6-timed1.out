Welcome to the Simple Expression Evaluator.
Enter one expression per line, end with ^D

   1: 1
   2: 1
   3: 2
   4: 4
   5: 8
   6: 16
   7: 32
   8: 64
   9: 27
  10: 54
  11: 7
  12: 14
  13: 28
  14: 56
  15: 11
  16: 22
  17: 44
  18: 88
  19: 75
  20: 49
  21: 98
  22: 95
  23: 89
  24: 77
  25: 53
  26: 5
  27: 10
  28: 20
  29: 40
  30: 80
  31: 59
  32: 17
  33: 34
  34: 68
  35: 35
  36: 70
  37: 39
  38: 78
  39: 55
  40: 9
  41: 18
  42: 36
  43: 72
  44: 43
  45: 86
  46: 71
  47: 41
  48: 82
  49: 63
  50: 25
  51: 3
  52: 75
  53: 49
  54: 98
  55: 95
  56: 89
  57: 77
  58: 53
  59: 5
  60: 10
  61: 20
  62: 40
  63: 80
  64: 59
  65: 17
  66: 34
  67: 68
  68: 35
  69: 70
  70: 39
  71: 78
  72: 55
  73: 9
  74: 18
  75: 36
  76: 72
  77: 43
  78: 86
  79: 71
  80: 41
  81: 82
  82: 63
  83: 25
  84: 50
  85: 100
  86: 99
  87: 97
  88: 93
  89: 85
  90: 69
  91: 37
  92: 74
  93: 47
  94: 94
  95: 87
  96: 73
  97: 45
  98: 90
  99: 79
 100: 57
 101: 13
 102: 26
 103: 52
 104: 3
 105: 6
 106: 12
 107: 24
 108: 48
 109: 96
 110: 91
 111: 
